Add sample search announcement to 'What's new'. (#7413)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/whatsnew/index.md

Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.

### Extension samples now searchable {: #extension-samples-searchable }

<p class="color-secondary-text type--caption">Posted on <time>September 29, 2023</time></p>

> We recently created a page that let's you [search Chrome extension samples](/docs/extensions/samples/). This is part of our ongoing effort to provide better answers and make them easier to find. The search page has multiple options. A search box lets you search for text in sample titles. You can restrict the search by permission or extension API. An additional filter lets you restrict search to either API or functional (use case) samples.

Reviewer: I think this is right?

```suggestion
We recently created a page that lets you [search Chrome extension samples](/docs/extensions/samples/). This is part of our ongoing effort to make sample code easier to find. The search page has multiple options. A search box lets you search for text in sample titles. You can restrict the search by permission or extension API. An additional filter lets you restrict search to either API or functional (use case) samples.
```

> This new sample page was build a Google Summer of Code participant, Xuezhou Dai, who also contributed new samples and fixed some of our bugs. You can read about his experiences this past summer in [his post on our blog](/blog/google-summer-of-code-and-chrome-extensions/).

Reviewer: ```suggestion
This new sample page was built by a Google Summer of Code participant, Xuezhou Dai, who also contributed several new samples. You can read about their experiences this past summer in [their post on our blog](/blog/google-summer-of-code-and-chrome-extensions/).
```
